docs/data-management.rst

```diff
@@ -226,6 +232,87 @@ Typically an administrator works on behalf of other users in a group where the a
 
 #. When creating new Datasets or Projects during the move, note that these containers will belong to the owner of the data, not yourself. Also the links between the new containers and the moved data will belong to the owner of the data. This should help to facilitate a smooth workflow, retaining the data handling possibilities such as reorganizing the data, renaming the containers you created for them etc. for the owner of the data. 
 
+.. _Duplicatecli:
+
+*Command Line: Duplicating objects*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can duplicate objects in OMERO. The functionality is available only on the CLI for now. The duplication creates a fully independent duplicate of the duplicated object and, when desired, the objects linked to the duplicated object as well. When the duplicate is later deleted, it will not have an influence on the original, which stays preserved. Similarly, when the original is later deleted, the duplicate stays preserved as well.
+
+In case of Image objects, which have image files linked, the duplication creates a new image file which is linked to the original image file by a hard link. This means every duplication of an Image increases the number of hard links on the image file in Managed Repository of OMERO, but does not duplicate the image file itself, and thus does not increase the storage demands too much. 
+
+In case of File Attachment objects though, which also have files to them, each duplication will duplicate the linked file, thus doubling the storage space necessary for these File Attachment files.
```

Comment:
Is this always the case? I thought a hard-link would be attempted first?

Reply:
Not for attachments because of the risk of a user copying someone else's then writing to it.

Reply:
This sounds like the formulation is acceptable (let me know if I get this wrong)

Reply:
I definitely stand (or sit) corrected, but I worry about the implications of this without control or feedback on how much space can be consumed.

Reply:
Certainly not a big step from here to add some space limit.

Reply:
I guess this again shows that we should get this doc in fairly quickly to be able to pass such granular info to the user

Reply:
:+1: